From 68c36e73e0009736a0b7be0505526d7069b8d4b9 Mon Sep 17 00:00:00 2001
Message-ID: <68c36e73e0009736a0b7be0505526d7069b8d4b9.1734811274.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1734811274.git.sharlatanus@gmail.com>
References: <cover.1734811274.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 20 Dec 2024 23:33:03 +0000
Subject: [PATCH 36/38] gnu: python-stdatamodels: Update to 2.2.0.

* gnu/packages/astronomy.scm (python-stdatamodels): Update to 2.2.0.
[arguments] <test-flags>: Skip one more test.
[propagated-inputs]: Remove python-asdf and python-psutil; add python-asdf-3.
[native-inputs]: Add nss-certs-for-test and python-psutil.

Change-Id: I557491aa84854b17a73d23598edb384576aa3133
---
 gnu/packages/astronomy.scm | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index d180ea4451..4c5a3cfda0 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -5319,13 +5319,13 @@ (define-public python-stcal
 (define-public python-stdatamodels
   (package
     (name "python-stdatamodels")
-    (version "2.1.2")
+    (version "2.2.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "stdatamodels" version))
        (sha256
-        (base32 "0x7fzkda9cr7bsf8hq2q090a73cxk3l3ynipq8lqixfp8c94m3kq"))))
+        (base32 "14720438p245ncjgwh7j37s69jd91rdk39daab8zswk9lhfbq9dx"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -5333,20 +5333,21 @@ (define-public python-stdatamodels
       #~(list "--numprocesses" (number->string (parallel-job-count))
               ;; Disable tests requiring access to CRDS servers to download
               ;; ~500MiB of data.
-              "-k" "not test_crds_selectors_vs_datamodel")
+              "-k" "not test_crds_selectors_vs_datamodel and not test_report")
       #:phases
       #~(modify-phases %standard-phases
           (add-before 'check 'set-home
             (lambda _
               (setenv "HOME" "/tmp"))))))
     (propagated-inputs
-     (list python-asdf
+     (list python-asdf-3
            python-asdf-astropy
            python-astropy
-           python-numpy
-           python-psutil))
+           python-numpy))
     (native-inputs
-     (list python-crds
+     (list nss-certs-for-test
+           python-crds
+           python-psutil
            python-pytest
            python-pytest-doctestplus
            python-pytest-xdist
-- 
2.46.0

